index.html

```diff
@@ -16,7 +16,50 @@
     <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <!-- ** Header ** -->
+    <header class="header">
+      <nav class="navbar">
+      <h1 class="title">CodeYourFuture News</h1>
+      <p class="desc" >CYF students are trained in full-stack web development by volunteers from the tech industry, putting a strong emphasis on collaboration and product development through tech projects.
+      </p>
+      </nav>
+    </header>
+     <!-- ** Header ** -->
+      <!-- ** Main ** -->
+    <main class="main">
+      <!-- ** Article ** -->
+      <article>
+      <section class="article">
+        <h2 id="css">CSS Week 1</h2>
+        <img class="articleImg" src="Img/css design.jpg" alt="css design">
```

Comment:
Camel case is great for JavaScript :) But for CSS, the convention is usually to hyphenate. Check this out: https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/